examples/tutorials/Introduction_to_Binding_Sites.ipynb

```diff
@@ -0,0 +1 @@
  321  NZ  LYS A 263      13.313  48.592  -6.636  1.00 83.28           N  \\nATOM    322  N   TYR A 264       9.178  54.639  -5.102  1.00 74.56           N  \\nATOM    323  CA  TYR A 264       7.865  54.775  -4.486  1.00 72.45           C  \\nATOM    324  C   TYR A 264       7.554  56.210  -4.067  1.00 70.42           C  \\nATOM    325  O   TYR A 264       6.547  56.465  -3.403  1.00 67.50           O  \\nATOM    326  CB  TYR A 264       7.758  53.846  -3.274  1.00 76.26           C  \\nATOM    327  CG  TYR A 264       8.166  52.418  -3.566  1.00 79.90           C  \\nATOM    328  CD1 TYR A 264       9.509  52.042  -3.564  1.00 82.83           C  \\nATOM    329  CD2 TYR A 264       7.214  51.450  -3.881  1.00 81.18           C  \\nATOM    330  CE1 TYR A 264       9.895  50.738  -3.870  1.00 83.07           C  \\nATOM    331  CE2 TYR A 264       7.588  50.143  -4.191  1.00 82.35           C  \\nATOM    332  CZ  TYR A 264       8.930  49.794  -4.184  1.00 83.29           C  \\nATOM    333  OH  TYR A 264       9.310  48.507  -4.499  1.00 84.17           O  \\nATOM    334  N   SER A 265       8.411  57.146  -4.466  1.00 68.74           N  \\nATOM    335  CA  SER A 265       8.224  58.553  -4.118  1.00 67.51           C  \\nATOM    336  C   SER A 265       7.960  58.633  -2.618  1.00 64.13           C  \\nATOM    337  O   SER A 265       7.168  59.452  -2.153  1.00 65.76           O  \\nATOM    338  CB  SER A 265       7.032  59.133  -4.882  1.00 69.42           C  \\nATOM    339  OG  SER A 265       7.171  58.920  -6.275  1.00 72.47           O  \\nATOM    340  N   LEU A 266       8.639  57.770  -1.872  1.00 58.48           N  \\nATOM    341  CA  LEU A 266       8.480  57.695  -0.429  1.00 53.91           C  \\nATOM    342  C   LEU A 266       9.614  58.376   0.328  1.00 51.41           C  \\nATOM    343  O   LEU A 266      10.793  58.124   0.062  1.00 50.02           O  \\nATOM    344  CB  LEU A 266       8.401  56.228  -0.006  1.00 51.74           C  \\nATOM    345  CG  LEU A 266       8.126  55.926   1.468  1.00 53.35           C  \\nATOM    346  CD1 LEU A 266       6.696  56.323   1.812  1.00 50.66           C  \\nATOM    347  CD2 LEU A 266       8.340  54.444   1.732  1.00 48.73           C  \\nATOM    348  N   THR A 267       9.252  59.247   1.266  1.00 47.07           N  \\nATOM    349  CA  THR A 267      10.247  59.930   2.081  1.00 44.50           C  \\nATOM    350  C   THR A 267      10.635  58.958   3.188  1.00 41.84           C  \\nATOM    351  O   THR A 267       9.771  58.382   3.848  1.00 40.72           O  \\nATOM    352  CB  THR A 267       9.691  61.212   2.708  1.00 45.02           C  \\nATOM    353  OG1 THR A 267       9.328  62.129   1.672  1.00 46.49           O  \\nATOM    354  CG2 THR A 267      10.736  61.856   3.606  1.00 40.98           C  \\nATOM    355  N   VAL A 268      11.935  58.779   3.386  1.00 37.58           N  \\nATOM    356  CA  VAL A 268      12.429  57.849   4.387  1.00 36.97           C  \\nATOM    357  C   V
```

Comment:
<p>Make these links to the docs (the classes to the docs)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4022/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>